pkg/upload/react-native-android.go

```diff
@@ -109,12 +110,18 @@ func ProcessReactNativeAndroid(options options.CLI, endpoint string, logger log.
 		}
 
 		if androidOptions.Android.AppManifest == "" {
-			appManifestPathExpected := filepath.Join(buildDirPath, "intermediates", "merged_manifests", androidOptions.Android.Variant, "AndroidManifest.xml")
+			appManifestPathExpected = filepath.Join(buildDirPath, "intermediates", "merged_manifests", androidOptions.Android.Variant, "AndroidManifest.xml")
 			if utils.FileExists(appManifestPathExpected) {
 				androidOptions.Android.AppManifest = appManifestPathExpected
 				logger.Debug(fmt.Sprintf("Found app manifest at: %s", androidOptions.Android.AppManifest))
 			} else {
-				logger.Debug(fmt.Sprintf("No app manifest found at: %s", appManifestPathExpected))
+				appManifestPathExpected = filepath.Join(buildDirPath, "intermediates", "merged_manifests", androidOptions.Android.Variant, "process"+androidOptions.Android.Variant+"Manifest", "AndroidManifest.xml")
```

Comment:
Given our expected case (`debug` vs `Debug`) this doesn't look like it'll work for case-sensitive filesystems?

Reply:
Correct. I've updated it so that in this case the variant gets capitalised.